sphinx/builders/linkcheck.py

```diff
@@ -312,6 +329,13 @@ def write_entry(self, what, docname, line, uri):
             output.write("%s:%s: [%s] %s\n" % (self.env.doc2path(docname, None),
                                                line, what, uri))
 
+    def write_json_entry(self, data):
```

Comment:
It would be better to rename "json entry" to another one. I think "json entry" does not describe what the data is.

Reply:
Changed to ``write_jsonline(linkstat)``

Reply:
Changed to `write_linkstat(linkstat)`

Reply:
(Though the other method is named `write_line`, so `write_jsonline` is logical and consistent)